tqsdk/zq.py

```
@@ -0,0 +1,44 @@
+                if sys.platform.startswith("win"):
+                    zq_proc = subprocess.Popen([interpreter, "-m", "tqsdk_zq.cli", "start"], stdin=subprocess.DEVNULL, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+                    returncode = await self._api._loop.run_in_executor(None, lambda: zq_proc.wait())
+                else:
+                    zq_proc = await asyncio.create_subprocess_exec(interpreter, "-m", "tqsdk_zq.cli", "start", stdin=subprocess.DEVNULL, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+                    returncode = await zq_proc.wait()
+                if returncode != 0:
+                    raise TqContextManagerError(f"无法拉起 zq 进程, 返回码: {returncode}")
+                return td_url
+        except (OSError, KeyError, json.JSONDecodeError):
+            raise Exception(f"加载配置文件失败: {self._zq_config_path}, 请尝试重新初始化: tqsdk-zq init") from None
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)